### PR TITLE
Add Start and End Period Field

### PR DESCRIPTION
**Pull request by steveoni**
This pr adds Start and End Period Field to create/edit Dataset #38 

Screenshot 2022-04-18 at 16 53 02

Screenshot 2022-04-18 at 16 54 59

### CODE REVIEW

File: ckanext/fcscopendata/assets/webassets.yml [hunk 1]
```diff
@@ -17,4 +17,16 @@ home_css:
   filter: cssrewrite
   output: ckanext-fcscopendata/%(version)s-home.css
   contents:
-    - home.css
\ No newline at end of file
+    - home.css
+
+
+# main-js:
```
Comment (sagargg) [inline ckanext/fcscopendata/assets/webassets.yml:23]: Let's remove these commented unnecessary lines.

File: ckanext/fcscopendata/logic/action.py [hunk 2]
```diff
@@ -57,6 +60,18 @@ def package_create(up_func, context, data_dict):
     data_dict['title'] =  data_dict.get('title_translated-en', '')
     data_dict['notes'] =  data_dict.get('notes_translated-en', '')
 
+    start_period = str(data_dict.get('start_period')) + "-01"
+    end_period = str(data_dict.get('end_period')) + "-01"
```
Comment (sagargg) [inline ckanext/fcscopendata/logic/action.py:64]: Does` 01` refers to days?  if that so,
 I think the end day should be the last day of the particular month. 
 eg. if selected April as the end month then the day should be 29.

**Comment by sagargg** [comment]
@steveoni  Could you also explain how i can search the datasets via the `package_search` action with the selected date range? I mean what will the query parameters look like?

**Comment by steveoni** [comment]
> @steveoni Could you also explain how i can search the datasets via the `package_search` action with the selected date range? I mean what will the query parameters look like?

http://ckan-dev:3000/api/action/package_search?fq=start_period:2022-12+end_period:2023-03

**Comment by sagargg** [comment]
LGTM 👍

**Pull request merged**